payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -253,20 +280,24 @@ fun PaymentsTheme(
 // This mirrors an object that lives inside of MaterialTheme.
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 object PaymentsTheme {
+    var colorsDarkMutable = PaymentsThemeDefaults.colorsDark
+    var colorsLightMutable = PaymentsThemeDefaults.colorsLight
     val colors: PaymentsComposeColors
         @Composable
         @ReadOnlyComposable
-        get() = PaymentsThemeConfig.toComposeColors()
+        get() = (if (isSystemInDarkTheme()) colorsDarkMutable else colorsLightMutable).toComposeColors()
```

Comment:
Will this work correctly when the colors are customized?

Reply:
It's been working in my testing. I'm seeing the colors for each mode selected and they change when I change the OS color mode.